modular_zubbers/code/game/machinery/burger_reactor/reactor.dm

```diff
@@ -358,6 +358,51 @@
 		vent_pressure_multiplier += new_servo.tier * 0.25
 	vent_pressure = initial(vent_pressure) * vent_pressure_multiplier
 
+/obj/machinery/power/rbmk2/ui_interact(mob/user, datum/tgui/ui)
+	ui = SStgui.try_update_ui(user, src, ui)
+	if(!ui)
+		ui = new(user, src, "RBMK2", name)
+		ui.open()
+
+/obj/machinery/power/rbmk2/ui_data(mob/user)
+	var/list/data = list()
+	data["active"] = active
+	data["rod"] = stored_rod
+	data["last_power_output"] = display_power(last_power_generation)
+	data["efficiency"] = power_efficiency*100
+	data["consuming"] = last_tritium_consumption*1000
```

Comment:
It'd be cool if you multiplied by this by 10000 instead of 1000 and then added the [μmol/s](https://en.wiktionary.org/wiki/%CE%BCmol) at the end of the "tritium consumption" UI element to indicate what the stat means

Reply:
Sure I will do it tomorrow

Reply:
> Sure I will do it tomorrow

did ya do it

Reply:
> > Sure I will do it tomorrow
> 
> did ya do it

Yes